proto/cosmos/authz/v1beta1/event.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+package cosmos.authz.v1beta1;
+
+// import "cosmos_proto/cosmos.proto";
+// import "google/protobuf/timestamp.proto";
+// import "gogoproto/gogo.proto";
+// import "google/protobuf/any.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/authz/types";
+
+
+// EventGrant is emitted on Msg/Grant
+message EventGrant {
+  // Module which invokes the authorizaiton grant
+  // TODO: not sure if this is needed. It always has the same value
+  string module = 1;
+  // Msg type for which an autorization is granted
+  string msg = 2;
```

Comment:
Should it be `msg_type_url` to be consistent?

Reply:
good point, yes, I will rename it.

Reply:
although `msg` also sounds good for events.

Reply:
I renamed to `msg_type_url`, but don't have preference. So if anyone else have some please comment.